src/data/index.js

```diff
@@ -1820,4 +1820,52 @@ export default [
       'Bloomberg',
     ],
   },
+  {
+    id: 164,
+    name: 'Longest Word in Dictionary',
+    url: 'https://leetcode.com/problems/longest-word-in-dictionary/',
+    pattern: 'Trie',
+    difficulty: 'Easy',
+    companies: [],
+  },
+  {
+    id: 165,
+    name: 'Maximum XOR of Two Numbers in an Array',
+    url: 'https://leetcode.com/problems/maximum-xor-of-two-numbers-in-an-array',
+    pattern: 'Trie',
+    difficulty: 'Medium',
+    companies: [],
+  },
+  {
+    id: 166,
+    name: 'Concatenated Words',
+    url: 'https://leetcode.com/problems/concatenated-words/',
+    pattern: 'Trie',
+    difficulty: 'Hard',
+    companies: [],
+  },
+  {
+    id: 167,
+    name: 'Prefix and Suffix Search',
+    url: 'https://leetcode.com/problems/prefix-and-suffix-search/',
+    pattern: 'Trie',
+    difficulty: 'Hard',
+    companies: [],
+  },
+  {
+    id: 168,
+    name: 'Palindrome Pairs',
+    url: 'https://leetcode.com/problems/palindrome-pairs/',
+    pattern: 'Trie',
+    difficulty: 'Hard',
+    companies: [],
+  },
+  {
+    id: 169,
```

Comment:
This is actually [#150](https://github.com/SeanPrashad/leetcode-patterns/pull/9/files#diff-4e7eab389364ff6d83587bf8a1078565R1629) in the list - could you please update the pattern [#150]there from `DFS` to `Trie`?

Once that is done, we can remove this section since it'll be considered a duplicate 👍🏽